### PR TITLE
WW-200 embedable discussions posts height and ellipsis

### PR DESCRIPTION
**Pull request by cormorando**

### CODE REVIEW

File: extensions/wikia/CommunityPage/styles/components/_discussions-module.scss [hunk 1]
```diff
@@ -13,6 +13,25 @@
 			font-size: $typescale-size-plus-2;
 			font-weight: bold;
 		}
+
+		.embeddable-discussions-post-detail-columns {
+			height: 280px;
```
Comment (rybmat) [inline extensions/wikia/CommunityPage/styles/components/_discussions-module.scss:18]: seba would say: "Magic number" ;)

File: extensions/wikia/CommunityPage/styles/components/_discussions-module.scss [hunk 2]
```diff
@@ -13,6 +13,25 @@
...
+			position: relative;
+
+			.embeddable-discussions-title {
+				-webkit-box-orient: vertical;
+				display: -webkit-box;
+				font-size: $typescale-size-plus-1;
+				-webkit-line-clamp: 2;
```
Comment (rybmat) [inline extensions/wikia/CommunityPage/styles/components/_discussions-module.scss]: I think, there was mixin for that in CP extension

Comment (lukaszkonieczny) [reply extensions/wikia/CommunityPage/styles/components/_discussions-module.scss]: Is it actually supported in Firefox?

File: extensions/wikia/CommunityPage/styles/components/_discussions-module.scss [hunk 3]
```diff
@@ -13,6 +13,25 @@
 			font-size: $typescale-size-plus-2;
 			font-weight: bold;
 		}
+
+		.embeddable-discussions-post-detail-columns {
+			height: 280px;
+			position: relative;
+
+			.embeddable-discussions-title {
+				-webkit-box-orient: vertical;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_discussions-module.scss]: Can we make mixin for this?

File: extensions/wikia/CommunityPage/styles/components/_discussions-module.scss [hunk 4]
```diff
@@ -13,6 +13,25 @@
...
+				font-size: $typescale-size-plus-1;
+				-webkit-line-clamp: 2;
+				overflow: hidden;
+			}
+		}
+
+		.embeddable-discussions-post-toolbar {
+			position: absolute;
+			bottom: 0;
+			width: 100%;
```
Comment (lukaszkonieczny) [inline extensions/wikia/CommunityPage/styles/components/_discussions-module.scss:31]: Property ordering

File: extensions/wikia/CommunityPage/styles/components/_discussions-module.scss (hunk 1 above)
Comment (cormorando) [reply extensions/wikia/CommunityPage/styles/components/_discussions-module.scss:18]: I'll put it in the spacing standarization ticket

**Review by lukaszkonieczny — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/EmbeddableDiscussions/scripts/templates.mustache.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
 /* jshint ignore:start */ define( 'embeddablediscussions.templates.mustache', [], function() { 'use strict'; return {
```
Comment (WikiaTech) [inline extensions/wikia/EmbeddableDiscussions/scripts/templates.mustache.js:1]: MINOR Reformat the code to have only one statement per line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AOneStatementPerLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
- MINOR 1 minor

Watch the comments in this conversation to review them.

File: extensions/wikia/CommunityPage/styles/components/_discussions-module.scss [hunk 6]
```diff
@@ -13,6 +13,22 @@
...
 			font-weight: bold;
 		}
+
+		.embeddable-discussions-post-detail-columns {
+			height: 280px;
+			position: relative;
+
+			.embeddable-discussions-title {
+				@include trim-to-line-number(2);
+				font-size: $typescale-size-plus-1;
```
Comment (dianafa) [inline extensions/wikia/CommunityPage/styles/components/_discussions-module.scss:24]: we use a conventon that we put newline after includes

Comment (sebastianmarzjan) [reply extensions/wikia/CommunityPage/styles/components/_discussions-module.scss:24]: (and I think it's enforced by the guidelines-repo hooks)

**Pull request merged**